README.md

```diff
@@ -2,51 +2,32 @@
+## Table of Contents (links)
 
-If you find a vulnerability that may affect live deployments -- for example, by exposing a remote execution exploit -- please send your report privately to security@ipfs.io. Please DO NOT file a public issue.
-
-If the issue is a protocol weakness that cannot be immediately exploited or something not yet deployed, just discuss it openly.
+- [Overview](#overview)
+- [How IPFS Works](#how-ipfs-works) ([Talks](#ipfs-talks)) ([Demos](#demos-and-examples))
+- [Current State of IPFS](#current-state-of-ipfs) ([Security Considerations](#security-issues))
+- [Get Started](https://ipfs.io/docs/getting-started) ([Installation](https://ipfs.io/docs/install), [Setup](https://ipfs.io/docs/getting-started#init-the-repo) & [Going Online](https://ipfs.io/docs/getting-started/#going-online))
+- [Specifications](https://github.com/ipfs/specs#ipfs-specs) ([Whitepapers](https://github.com/ipfs/papers))
+- [Language Implementations](#language-implementations)
+- [Client Libraries](#client-libraries)
+- [Community Projects](#community-projects)
+- [FAQs: Frequently Asked Questions](#faq)
+- [Contributing](CONTRIBUTE.md) & [Roadmap](https://github.com/ipfs/pm) [![Ready](https://badge.waffle.io/ipfs/ipfs.png?label=ready&title=Backlog)](https://waffle.io/ipfs/ipfs)
 
-## [Project directory](project-directory.md)
-Lost? Searching for something? Need to submit an issue? Want to contribute? Take a peek at [project-directory.md](project-directory.md) which maps out all the repositories for the ipfs project on Github.
+## Overview
 
+IPFS ([InterPlanetary File System](#why-the-name-ipfs)) is an Open Source distributed file system that seeks to connect all computing devices with the same system of files. In some ways, this is similar to the original aims of the Web, but IPFS is actually more similar to a single bittorrent swarm exchanging git objects. You can read more about its origins in the paper [IPFS - Content Addressed, Versioned, P2P File System](https://github.com/ipfs/ipfs/blob/master/papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf?raw=true).
 
-## IPFS Talks
+IPFS could become a new major subsystem of the internet. If built right, it could complement or replace HTTP. It could complement or replace even more. It sounds crazy. It _is_ crazy.
 
 [![Alpha Demo](img/ipfs-alpha-video.png)](https://www.youtube.com/watch?v=8CMxDNuuAiQ)
 
-[![IPFS Talk](img/ipfs-talk.png)](https://www.youtube.com/watch?v=Fa4pckodM9g)
-
-
-## Design
-
-Please Read:
-
-- [IPFS - Content Addressed, Versioned, P2P File System](https://github.com/ipfs/ipfs/blob/master/papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf?raw=true)
-- IPFS - Toward The Permanent Web
+Want to see more? Check out Juan Benet's talk at Sourcegraph - [IPFS: The Permanent Web](https://www.youtube.com/watch?v=Fa4pckodM9g).
```

Comment:
I think we should link to https://www.youtube.com/watch?v=HUVmypx9HGI instead here